src/platforms/javascript/common/install/loader.mdx

```diff
@@ -84,42 +94,68 @@ You can configure the release by adding the following to your page:
+### Custom Configuration
+
+The loader script always includes a call to `Sentry.init` with a default configuration, including your DSN. If you want to <PlatformLink to="/configuration/options/">configure your SDK</PlatformLink> beyond that, you can configure a custom init call by defining a `window.sentryOnLoad` function. Whatever is defined inside of this function will _always_ be called first, before any other SDK method is called. Be sure to define this function _before_ you add the loader script, to ensure it can be called at the right time:
 
 ```html
+<script>
+  // Configure sentryOnLoad before adding the Loader Script
+  window.sentryOnLoad = function () {
+    Sentry.init({
+      // add custom config here
+    });
+  };
+</script>
+
 <script
   src="https://js.sentry-cdn.com/___PUBLIC_KEY___.min.js"
   crossorigin="anonymous"
 ></script>
-<script>
-  // Check for existence of Sentry in case Ad-blockers block the Sentry Loader Script
-  window.Sentry && Sentry.onLoad(function() { ... });
-</script>
 ```
 
-`Sentry.onLoad` is a function that only the loader provides, and - as the name suggests - it sets a function to be run once the full SDK has been loaded. In that function, you can configure your SDK exactly the way you would were you using the CDN, with one difference: your `Sentry.init` call doesn't need to include your DSN, since it's already been set. We recommend checking if `Sentry` exists before calling `Sentry.onLoad` in case the loader script failed to load.
+Inside of the `window.sentryOnLoad` function, you can configure a custom `Sentry.init()` call. You can configure your SDK exactly the way you would if you were using the CDN, with one difference: your `Sentry.init()` call doesn't need to include your DSN, since it's already been set. Inside of this function, the full Sentry SDK is guaranteed to be loaded & available.
 
 ```html
 <script>
-  // Check for existence of Sentry in case Ad-blockers block the Sentry Loader Script
-  window.Sentry && Sentry.onLoad(function() {
+  // Configure sentryOnLoad before adding the Loader Script
+  window.sentryOnLoad = function () {
     Sentry.init({
       release: " ... ",
       environment: " ... "
     });
-    Sentry.configureScope(scope => {
-      scope.setTag( ... );
-    });
+    Sentry.setTag(...);
     // etc.
-  });
+  };
 </script>
 ```
 
-For Performance Monitoring, by default, the SDK will be initialized with `tracesSampleRate: 1`. This means that the SDK will capture all traces.
+### Guarding SDK Function Calls
 
-For Session Replay, the defaults are set to `replaysSessionSampleRate: 0.1` and `replaysOnErrorSampleRate: 1`. This means Replays will be captured for 10% of all normal sessions and for all sessions with an error.
+The loader will by default make sure that these functions can be called directly on `Sentry` at all times, even if the SDK is not yet loaded:
```

Comment:
My 2 cents - this section seems pretty clear to me now. I did make one small wording suggestion to make this sentence flow better.

```suggestion
By default, the loader will make sure you can call these functions directly on `Sentry` at any time, even if the SDK is not yet loaded:
```